apps/Launcher/qml/style.js

```diff
@@ -33,6 +33,7 @@ var fileBrowserDiscreteTextColor = "#B0B0B0"
 var fileBrowserBlinkColor = "lightblue"
 var fileBrowserItemSizeRel = 0.2
 var fileBrowserTextSizeRelToItem = 0.09
+var fileBrowserGridSectionTextRelSize = 0.35
```

Comment:
I try to add the most important numbers here but it's cumbersome to do it for every single value (such as a 2x or 0.5) so it's true that I don't do it when it's a very localized style issue. Sometimes it is also easier to look directly in the qml to change minor style details rather than having to check if a constant is used in multiple places...